src/parser/shared/modules/Entities.ts

```diff
@@ -125,6 +123,38 @@ abstract class Entities<T extends Entity> extends Analyzer {
     }
   }
 
+  refreshBuff(event: RefreshBuffEvent | RefreshDebuffEvent) {
```

Comment:
I ended up not using this addition to Entities, but its harmless and may be useful in the future so I left it in